how-to/features/tls-vault.rst

```diff
@@ -0,0 +1,69 @@
+TLS Vault
+=========
+
+This feature is used to encrypt all cloud service endpoints (both public
+and private) using TLS certificates generated by Vault, which acts as an intermediary CA.
+It does this by interfacing with the existing Traefik instances in the
+cloud. A Traefik instance is associated with either public or private
+cloud traffic.
+
+Prerequisites
+-------------
+
+To use TLS Vault, you must enable the Vault feature in your cloud and unseal and authorize the Vault charm.
+This is done by following guide :doc:`Enable Vault </how-to/features/vault>`.
+
+Enable TLS Vault
+----------------
+
+To enable TLS Vault, you’ll need to provide information that identifies your
+chosen Certificate Authority. Do this by specifying a CA certificate and
+its CA certificate chain.
+
+Run the following command to enable TLS Vault for public endpoints:
+
+::
+
+   sunbeam enable tls vault --ca <base64 encoded ca certificate> --ca-chain <base64 encoded ca chain>
+
+To enable TLS Vault for public, internal and rgw endpoints, be explicit by
+using the ``--endpoint`` option:
+
+::
+
+   sunbeam enable tls vault --ca <base64 encoded ca certificate> --ca-chain <base64 encoded ca chain> --endpoint public --endpoint internal --endpoint rgw
+
+Disable TLS Vault
```

Comment:
This should go to the very end, after the "Use TLS Vault" section. "Cleaning up resources" sections should come at the end because a user would typically want to disable the feature after they have actually used it.

Reply:
@hemanthnakkina agreed with @skoech, should I also update the TLS CA to move the disable part to the end?